tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -15,6 +15,7 @@
 #include <iostream>
 #include <list>
 #include <map>
+#include <numeric>

```

Comment:
Although this PR is implementing the Spaceship changes for Clause 26 Numerics, there are no implementation changes to `<numeric>` (the numeric algorithms header), so it's unnecessary to include. Only `<valarray>` is needed for the test code below.